docs/DeveloperGuide.md

```diff
@@ -256,14 +257,18 @@ _{Explain here how the data archiving feature will be implemented}_
 ### Product scope
 
 **Target user profile**:
+The targets users...
```

Comment:
Should this be _target_ users?

Reply:
Thanks for checking! I have rectified the error, can you check again?